src/components/movies/MovieList.vue

```diff
@@ -0,0 +1,66 @@
+<script>
+import { defineComponent } from 'vue';
+
+import MovieListCard from '@/components/movies/MovieListCard.vue';
+
+export default defineComponent({
+  props: {
+    movies: {
+      type: Array,
+      required: true,
+    },
+  },
+  components: {
+    MovieListCard,
+  },
+  computed: {
+    movieCardClasses() {
+      return {
+        'movie-list__card': true,
+        'movie-list__card--homepage-view': this.movies.length <= 3,
+      };
```

Comment:
A small tip 💡  - if you do not like the `className: true` part you can actually do it differently to omit it!

```suggestion
      return [
        'movie-list__card',
        {
          'movie-list__card--homepage-view': this.movies.length <= 3,
        }
      ];
```

Reply:
Actually it's the other way around 😄
I like it to be on one level of indentation and this little `true` keyword doesn't bother me at all 😄